geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/ExportCommandTools.scala

```diff
@@ -0,0 +1,90 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo.commands
+
+import java.io._
+import java.util.zip.GZIPOutputStream
+import scala.collection.JavaConversions._
```

Comment:
Nope, I just didn't know there was an 'organize inputs' command. I just figured built in libs came first.